k2/csrc/properties.h

```diff
@@ -1,10 +1,14 @@
-#include "k2/fsa.h"
+#ifndef K2_PROPERTIES_H_
+#define K2_PROPERTIES_H_
+
+#include <stdint.h>
+#include "fsa.h"
 
 
 namespace k2 {
 
 // TODO: we might not need this.
-enum {
+enum Properties{
   kTopSorted,  // topologically sorted
   kTopSortedAndAcyclic,  // topologically sorted and no self-loops (which implies acyclic)
```

Comment:
The comment for `kTopSortedAndAcyclic` is not accurate.

Acyclic implies no self-loops.

But no self-loops does not mean it is acyclic, e.g, 0 -> 1 ->2 -> 0, it has no self-loops, but there is a cycle.

Reply:
together they imply acyclic.. guess you realized this.  perhaps "implies" could be reworded to "imply" which makes it clearer it's both things on the left together which imply the thing on the right.